Classes/Views/DotListView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DotView.swift
+//  Freetime
+//
+//  Created by Joe Rocca on 10/19/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class DotListView: UIView, UICollectionViewDataSource {
+    
+    private let collectionView: UICollectionView
+    private var colors = [UIColor]()
+    
+    init(frame: CGRect = .zero,
+         dotSize: CGSize = CGSize(width: 10, height: 10)) {
+        
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        layout.itemSize = dotSize
+        layout.minimumLineSpacing = Styles.Sizes.columnSpacing / 2.0
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.backgroundColor = .clear
+        collectionView.isUserInteractionEnabled = false
+        collectionView.register(IssueLabelDotCell.self, forCellWithReuseIdentifier: IssueLabelDotCell.reuse)
+        
+        super.init(frame: frame)
+        
+        collectionView.dataSource = self
+        addSubview(collectionView)
+        collectionView.snp.makeConstraints { (make) in
+            make.edges.equalTo(self)
+        }
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        collectionView.reloadData()
```

Comment:
Can we remove this? It isn't necessary is it? Overkill.

Reply:
This is actually here so the the max items are recalculated on orientation change.

![dots](https://user-images.githubusercontent.com/10346184/31857879-c6c1f238-b6b6-11e7-897c-37d44755e9bc.gif)